infra/feast-operator/internal/controller/services/services.go

```diff
@@ -474,6 +476,26 @@ func (feast *FeastServices) getServiceConfigs(feastType FeastServiceType) feastd
 	return feastdevv1alpha1.ServiceConfigs{}
 }
 
+func (feast *FeastServices) getLogLevelForType(feastType FeastServiceType) string {
```

Comment:
I think the current design uses `util.ApplyDefaultsToStatus` function to generate the default value under `Status.Spec.Applied` field, instead of calculating it on the fly when it is needed. @tchughesiv ?

Reply:
this will not be a "default" type of config in the status